### PR TITLE
Add test for choice with no vrCommands, fix typo in file

### LINKED ISSUE #1988: add testing for choice VR optional
Need testing for functionality provided by https://github.com/smartdevicelink/sdl_core/issues/2160 to test https://github.com/smartdevicelink/sdl_core/pull/2415

### PR DESCRIPTION
**Pull request by conlain-k**
fixes #1988, also fixes choise/choice typo while we're here

### CODE REVIEW

File: test_scripts/Smoke/API/012_PerfomInteraction_PositiveCase_SUCCESS.lua [hunk 1]
```diff
@@ -91,13 +91,27 @@ local requestParams = {
   interactionLayout = "ICON_ONLY"
 }
 
+local requestParams_noVR = {
```
Comment (GetmanetsIrina) [inline test_scripts/Smoke/API/012_PerfomInteraction_PositiveCase_SUCCESS.lua]: requestParams_noVR is not used in script, propose that must be in line 401 .

File: test_scripts/Smoke/API/012_PerfomInteraction_PositiveCase_SUCCESS.lua [hunk 2]
```diff
@@ -114,6 +128,24 @@ local function setChoiseSet(choiceIDValue)
   return temp
 end
 
+--! @setChoiceSet: Creates Choice structure without VRcommands
```
Comment (GetmanetsIrina) [inline test_scripts/Smoke/API/012_PerfomInteraction_PositiveCase_SUCCESS.lua]: @setChoiceSet must be matched to function name -> @setChoiceSet_noVR

File: test_scripts/Smoke/API/012_PerfomInteraction_PositiveCase_SUCCESS.lua (hunk 1 above)
Comment (conlain-k) [reply test_scripts/Smoke/API/012_PerfomInteraction_PositiveCase_SUCCESS.lua]: You are correct, I have updated line 401 to match. Thank you!

File: test_scripts/Smoke/API/012_PerfomInteraction_PositiveCase_SUCCESS.lua (hunk 2 above)
Comment (conlain-k) [reply test_scripts/Smoke/API/012_PerfomInteraction_PositiveCase_SUCCESS.lua]: Updated!

**Pull request merged**